docker/build-tools/Dockerfile

```diff
@@ -152,14 +150,17 @@ ADD https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${H
 RUN tar -xzvf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz -C /tmp
 RUN mv /tmp/hugo ${OUTDIR}/usr/bin
 
-# Helm and helm3
-ADD https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz /tmp
-RUN tar -xf /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz -C /tmp
-RUN mv /tmp/linux-amd64/helm ${OUTDIR}/usr/bin
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN curl -L ${HELM3_RELEASE_ROOT}/${HELM3_RELEASE_FILE} |tar xvz && \
-    mv linux-amd64/helm /usr/bin/helm3 && \
-    chmod +x /usr/bin/helm3
+# Helm version 2
+ADD https://storage.googleapis.com/kubernetes-helm/helm-${HELM2_VERSION}-linux-amd64.tar.gz /tmp
+RUN mkdir /tmp/helm2
```

Comment:
if we call this helm2 its going to break everything referring to `helm`? It seems to have precedence to keep the legacy one unversioned ( `python` and `python3` ), should we just do that?

Reply:
On line 157, the binary name is `helm` and is copied directly into the OUTDIR.

The name helm2 is just used for packaging. This is typically how packagers refer to (in your example) python2 internally in packaging in distros such as Debian, Ubuntu, Fedora, CentOS - versus what ends up on disk (line 157). Happy to sed helm2->helm, although I think it may be confusing for future maintainers.

Reply:
oops, missed that. Thanks